cli/pkg/config_manager/config_manager.go

```diff
@@ -69,10 +81,21 @@ func CreeateHappyConfig(ctx context.Context, bootstrapConfig *config.Bootstrap)
 		Help:    "This will be the unique name of the application, lowercased and hyphenated",
 		Default: filepath.Base(bootstrapConfig.HappyProjectRoot),
 	}
-	err = survey.AskOne(prompt1, &appName, survey.WithValidator(survey.Required), survey.WithValidator(survey.MinLength(5)))
+	err = survey.AskOne(prompt1, &appName,
+		survey.WithValidator(survey.Required),
+		survey.WithValidator(survey.MinLength(5)),
+		survey.WithValidator(survey.MaxLength(15)))
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to prompt")
 	}
+
+	appName = strings.TrimSpace(strings.ToLower(appName))
+	reg, err := regexp.Compile(`[^a-zA-Z0-9_]+`)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to compile regex")
+	}
+	appName = reg.ReplaceAllString(appName, "_")
```

Comment:
Should this use hyphen instead?
```suggestion
	appName = reg.ReplaceAllString(appName, "-")
```

Reply:
Existing example apps  use underscores for app name, we need to figure out how to update them.

Reply:
Is this just for net new apps?